crates/kornia-imgproc/src/features.rs

```diff
@@ -82,6 +93,179 @@ pub fn hessian_response(src: &Image<f32, 1>, dst: &mut Image<f32, 1>) -> Result<
     Ok(())
 }
 
+/// A builder object to initialize Harris response
+pub struct HarrisResponse {
+    image_size: ImageSize,
+    k: f32,
+    grads_mode: GradsMode,  // TODO
+    sigmas: f32,  // TODO
+    dx2_data: Vec<f32>,
+    dy2_data: Vec<f32>,
+    dxy_data: Vec<f32>,
+}
+
+impl HarrisResponse {
+    /// Creates a Harris response object with default values
+    pub fn new(image_size: ImageSize) -> Self {
+        Self {
+            image_size,
+            k: 0.04,
+            grads_mode: GradsMode::default(),
+            sigmas: 0.0,
+            dx2_data: vec![0.0; image_size.width*image_size.height],
+            dy2_data: vec![0.0; image_size.width*image_size.height],
+            dxy_data: vec![0.0; image_size.width*image_size.height],
+        }
+    }
+
+    /// Sets the `k` value (usually between 0.04 to 0.06)
+    pub fn with_k(self, k: f32) -> Self {
```

Comment:
can you add a small test to validate that you can concatenate calls in the builder pattern ? I'm curious to see how it works as I had implemented slightly different in here https://github.com/kornia/kornia-rs/blob/main/crates/kornia-io/src/stream/v4l2.rs#L47 maybe we should stick a common style